main/getting-started/intro-zoe.md

```diff
@@ -1,358 +1,286 @@
 # Introduction to Zoe
 
-**Note:** Before reading this, you should be familiar with [ERTP basics](/getting-started/ertp-introduction/). 
-
 ::: tip Alpha status
 Zoe is currently at Alpha status. It has not yet been
 formally tested or hardened. It is not yet of production quality.
 :::
 
-This is an introduction to Zoe, Agoric's smart contract framework. 
-The [Zoe Guide](/zoe/guide/) has additional information, 
-including concepts not covered here. 
+## What is Zoe?
 
-You can run different types of smart contracts on Zoe. In this document, 
-for the most part we focus on Agoric’s Atomic Swap demo contract, but 
-sometimes refer to how Zoe works with other contract types.
+Zoe is Agoric's smart contract framework. You should use Zoe to:
 
-## What is Zoe?
+* **Run your code on-chain**
+* **Mint new digital assets**
+* **Credibly trade assets**
+
+Zoe relies on [ERTP](./ertp-introduction.md), our token standard for fungible and non-fungible
+tokens. 
+
+## Why should I use Zoe?
+
+### For Users ###
+
+**Zoe is safer.** Putting digital assets in a smart contract has
+traditionally meant you might lose them. But Zoe guarantees that you
+will either get what you wanted or get a full refund. You will never
+leave a contract empty-handed, even if the smart contract is buggy or
+malicious. 
+
+### For Developers ###
+
+**Zoe is easier.**  Writing a smart contract has traditionally meant
+learning a completely new, untried language. And let's hope you don't
+make any mistakes - if you do, your users might lose millions.
+
+Zoe smart contracts, on the other hand, are written in a secure subset
+of JavaScript. Moreover, Zoe handles all of the escrowing of digital
+assets provided by your users and the subsequent payout of digital
+assets to your users. **Your contract, even if it's buggy, can't cause
+your users to lose their assets.**
+
+## How do I use Zoe?
+
+There are three ways you can get started using Zoe:
+
+* you can write a new contract and install it on Zoe
+* you can create a new `instance` of a contract already installed on
+  Zoe
+* you can receive an `invitation` to join a contract
+  instance that someone else has started.
+
+Agoric has written [a number of example contracts
+that you can use](/zoe/guide/contracts/), including:
+* a [Uniswap
+implementation](/zoe/guide/contracts/multipoolAutoswap.md)
+* a [covered
+call option contract](/zoe/guide/contracts/covered-call.md)
+* an [OTC
+Desk market maker
+contract](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/otcDesk.js)
+* contracts for [minting fungible](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/mintPayments.js) and [non-fungible tokens](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/mintAndSellNFT.js)
+
```

Comment:
Gotcha. I removed the first bit about the three ways. I think we do want people to know about the kind of contracts that can be built on Zoe, with links, early. The reader likely knows what Uniswap is, so that we have something like that, that they would be able to build themselves, is meaningful. And, the links are social proof that we're not talking abstractly and promising more than we can deliver. This is important to a dev audience who has been burned on other hyped projects.

"Example Agoric-written contracts for you to look at and use as templates" isn't right from a product perspective. We want to emphasize composability. That means you'll actually use these contracts - not that you will copy and paste and make your own. Using the term `template` gives the wrong idea. Of course, if anyone writes something new, they will look at other contracts, but we don't need to undermine our composability argument.

The Zoe links are correct. If you start with a `/` it starts from the root, which is `main`.